nfc: remove FITS references from README

elidana opened the pull request:
Hi @rumachan ,

there are no FITS tutorials I can see pending, but the README were still referencing to it.

rumachan reviewed (changes requested):
Simply remove " These data are currently delivered by [FITS](https://fits.geonet.org.nz/api-docs/) but will move to Tilde" from the GNSS entry in Volcano/README.md. Don't edit it. None of the other entries are written that way.

rumachan commented:
Tutorials are a pain in the but because you have to change both the web site page and the github page.

elidana commented:
Hi @rumachan , sorry I'm feeling really dumb as I don't understand your suggestion. 

The GeoNet website is already updated as far as I can tell (https://www.geonet.org.nz/data/access/tutorials) and there's no reference to FITS tutorial.

The changes I did here are just 
- on the main `README.md` file of this `data-tutorials` repository, where I think we forgot to remove the link to "FITS" folder (and if you click on it from https://github.com/GeoNet/data-tutorials?tab=readme-ov-file) it actually returns a "Page not found. So I'm not sure what would be the benefit of keeping that line
- on the `volcano/README.md` file , where I changed the info about data source for GNSS type and changed this from fits to tilde

or am I missing something super obvious?

elidana commented:
gosh ok sorry, totally overinterpreted your comment! :facepalm: 

have reworded (and removed the tilde reference, as it is done for the other data types.

@rumachan Can you please review again so we can merge? Thanks!!
